thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/AnomalyResource.java

```diff
@@ -495,9 +495,11 @@ public Response applyAutotuneConfig(@PathParam("autotuneConfigId") @NotNull long
     // if auto tune method is ALERT_FILTER_LOGISITC_AUTO_TUNE or INITIATE_ALERT_FILTER_LOGISTIC_AUTO_TUNE, alert filter is to be updated
     if (autotuneConfigDTO.getAutotuneMethod() != EXHAUSTIVE) {
       targetFunction.setAlertFilter(autotuneConfigDTO.getConfiguration());
-    } else{
+    } else {
       // Update function configuration
-      targetFunction.updateProperties(autotuneConfigDTO.getConfiguration());
+      if (autotuneConfigDTO.getConfiguration()!= null && !autotuneConfigDTO.getConfiguration().isEmpty()) {
```

Comment:
if new autotuneConfigDTO is empty, are we designed to not update target Function? Let's log this difference, the original function setting is already optimal after tuning.